src/mixins/_adjacent.scss

```diff
@@ -0,0 +1,29 @@
+@use "../compiled/tokens/scss/size";
```

Comment:
Should this be included in the changeset? Or is that not necessary since this is just an internal change? 🤔 

It's probably fine as-is, but figured I'd ask since I wasn't sure.

Reply:
I don't think so since it's just an internal change.